BUILDGUIDE.md

```diff
@@ -319,6 +319,12 @@ TLS 1.3 has been excluded due to the fact that the driver lacks full support. To
 
 `Switch.Microsoft.Data.SqlClient.EnableSecureProtocolsByOS`
 
+## Suppressing TLS security warning
+
+When connecting to a server, if a protocol lower than TLS 1.2 is negotiated, a security warning is output to the console. This warning can be suppressed on SQL connections with `Encrypt = false` by enabling the following AppContext switch on application startup:
```

Comment:
```suggestion
When connecting to a Sql server, if a protocol lower than TLS 1.2 is negotiated, a security warning is printed to the console. This warning can be suppressed on SQL connections while `Encrypt = false` by enabling the following AppContext switch on application startup:
```

Reply:
This is a backport, and we usually keep them identical with the source on the main branch. You can prepare a PR on the main branch If you think we need improvement here and backport it after getting merged.